crates/burn-dataset/Cargo.toml

```diff
@@ -45,6 +45,7 @@ gix-tempfile = { workspace = true, optional = true }
 globwalk = { workspace = true, optional = true }
 hound = { workspace = true, optional = true }
 image = { workspace = true, optional = true }
+polars = {version =  "0.41.3", features = ["polars-io"] }
```

Comment:
I would make this dependency as optional, so people using dataset won't have to download polars.